frontend/web/components/modals/CreateMetadataField.tsx

```diff
@@ -88,9 +86,11 @@ const CreateMetadataField: FC<CreateMetadataFieldType> = ({
   const [updateMetadataModelField] = useUpdateMetadataModelFieldMutation()
 
   const [deleteMetadataModelField] = useDeleteMetadataModelFieldMutation()
-  const projectContentType: ContentType =
-    supportedContentTypes &&
-    Utils.getContentType(supportedContentTypes, 'model', 'project')
+  const contentTypeKey = 'organisation'
```

Comment:
Do you think it makes sense to make `'organisation'` a constant [here]?(https://github.com/Flagsmith/flagsmith/blob/main/frontend/common/constants.ts)

Reply:
You're right, it adds no value here. I started an implem passing the scope (project or organisation) but as it's not implemented now I removed so it's properly done once needed.

Just pushed using an enum. I left it in the file, is there any other location that would make more sense ?